src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -895,7 +906,20 @@ def main(argv):
     arguments.append(constants.VALUE_PROFILE_ARGUMENT)
     fuzzing_strategies.append(strategy.VALUE_PROFILE_STRATEGY)
 
-  if do_fork():
+  use_dataflow_tracing = False
```

Comment:
Just initialize it to "dataflow_build_path and do_dataflow_tracing()", use in if, rather than setting to true later and initing to false ?

Reply:
Good idea, thanks, done!